README.md

```diff
@@ -1,5 +1,65 @@
-# Capstone-Project-2---API-based-webapp
-This repository is about building a web application based on an external API. We will select an API that provides data about a topic that we like and then build the webapp around it. The webapp will have 2 or 3 user interfaces. A Home Page, a popup window with more data and a reservation window. 
+# Capstone - Project-2 --- API Based Webapp
 
-# Github Toke
-This token will be used to get GitHub API instead of password: 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
\ No newline at end of file
+> This is a group project of three people <a href="https://github.com/for-ashraf">Muhammad Ashraf</a>, <a href="https://github.com/AbrahaKahsay">Abraha</a> and <a href="https://github.com/lucabenaventew">Luca</a>.
+
+> This project is about about building your own web application based on an external API. We will select an API that provides data about a topic that we liked and then build the webapp around it.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- webpack
+- Linters
+- API Consumption
+- Kanban Board
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps:
+
+- Run "npm start" command at terminal to browse the page.
+## Loom Video Link for a brief intro of the app
+
+- https://www.loom.com/share/873c948d2210470780a12d503a22e857
+
```

Comment:
- [ ] In the video presentation, Please show up all the members and talk about the project.